ably/rest/channel.py

```diff
@@ -27,21 +28,54 @@ def __init__(self, channel):
         self.__binary = not channel.ably.options.use_text_protocol
         self.__http = channel.ably.http
 
-    def get(self):
-        path = '%s/presence' % self.__base_path
+    def _path_with_qs(self, rel_path, qs=None):
+        path = rel_path
+        if qs:
+            path += ('?' + urlencode(qs))
+        return path
+
+    def _ms_since_epoch(self, dt):
+        epoch = datetime.utcfromtimestamp(0)
+        delta = dt - epoch
+        return int(delta.total_seconds() * 1000)
+
+    def get(self, limit=None):
```

Comment:
Sorry, my mistake, I didn't see that the Presence class is in fact defined in the Channel.py file.  Is there any reason this is not in a separate `Presence.py` file?